app/models.py

```diff
@@ -186,3 +191,35 @@ def is_admin(self):
 @login_manager.user_loader
 def load_user(user_id):
     return User.query.get(int(user_id))
+
+
+# Agency schema
```

Comment:
Nit: if you are going to comment about a class use docstrings like this http://stackoverflow.com/questions/3051241/how-to-document-class-attributes-in-python?rq=1